ex4/server/routes/api.js

```diff
@@ -0,0 +1,39 @@
+const ItemManager = require('../services/item_manager');
+
+async function getTodos(req, res) {
+	let data = await ItemManager.getTodos();
+	if (!data) data = [];
+
+	res.header('Access-Control-Allow-Origin', '*');
```

Comment:
i saw you already had it in several places including some middleware, why you needed this in all those places?